API/hermes/hermes.h

```diff
@@ -188,12 +188,19 @@ class HERMES_EXPORT HermesRuntime : public jsi::Runtime {
   /// Unregister this runtime for sampling profiler.
   void unregisterForProfiling();
 
+  /// Define methods to interrupt JS execution and set time limits.
+  /// All JS compiled to bytecode via prepareJS, or evaluateJS, will support interruption and
+  /// time limit monitoring.  If JS prepared in other ways is executed, care
```

Comment:
Turns out that this isn't true, which is why the tests in release mode are failing in CircleCI. Async break check is actually disabled by default, and in release builds, with debugging disabled, they are not emitted at all.

We should fix this, but I don't want to block your PR on it, so let's make the requirements clear.
```suggestion
  /// All JS compiled to bytecode via prepareJS, or evaluateJS, will support interruption and
  /// time limit monitoring if the runtime is configured with AsyncBreakCheckInEval.  If JS prepared in other ways is executed, care
```